### DISCUSSION TITLE
URGENT: Domain blacklisted AGAIN for using Simple Login

### DISCUSSION THREAD

**Discussion by SolluSaaS** [simple-login/app#2200] [Q&A]
Good evening,

This is the second time that one of my domains configured for use with SimpleLogin has been identified as "disposable" and has been blocked on blacklists.

First, it was the domain "loginseguro.online," which was wrongly blocked (I managed to unblock it after a lot of hassle), and now the domain "infraestrutura.online" is currently blocked.

Both blocks have caused me countless problems, including financial losses and damage to my reputation with clients and suppliers.

I believe that as a Premium customer, who trusts the service and the company, I should be entitled to a service that is more respected among the blacklist systems. I understand that some people misuse the service for malicious purposes, but I am JUST using the service to AVOID having my emails targeted by SPAM or malicious emails.

In the end, even though I am using the services correctly, I am treated like a cybercriminal for using SimpleLogin’s service.

I would like to ask for your help and guidance to intervene with "https://abuse.radix.website/unsuspension" to help me deal with this block, as it occurred because, according to them, I am using a "disposable email" service, which raises concerns about malicious use of the domain.

I urgently await your assistance as I am already facing issues with clients AGAIN due to this wrongful block generated by the use of the SimpleLogin tool!

Thank you very much.

**Comment by mlec1** [comment]
@SolluSaaS You might be more successful by contacting their support (never had to use it though) https://app.simplelogin.io/dashboard/support or [support@simplelogin.zendesk.com](mailto:support@simplelogin.zendesk.com)

**Comment by hiccups1980** [comment]
You should contact SL via their email address. We can't help you here.
https://simplelogin.io/docs/report-blocking-website/
